client/services/FeedbackGeneratorService.js

```diff
@@ -222,6 +225,43 @@ tie.factory('FeedbackGeneratorService', [
         feedback.setSyntaxErrorIndex(feedback.getParagraphs().length - 1);
         return feedback;
       },
+      getPrereqFailureFeedback: function(codePrereqCheckResult) {
+        var feedback = FeedbackObjectFactory.create(false);
+        var preReqFailures = codePrereqCheckResult.getPrereqCheckFailures();
```

Comment:
Here and elsewhere: be careful of casing. "Prerequisite" is one word, and so this should be `prereqFailures` (or, for parity with the method name, `prereqCheckFailures`).